modules/monitoring-common-data-types-for-monitoring-configuration.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc
+
+:_content-type: CONCEPT
+[id="common-data-types-for-monitoring-configuration_{context}"]
+= Common data types for monitoring configuration
+
+This section describes a set of common data types common for configuring both core platform monitoring components and components that monitor user-defined projects.
```

Comment:
IBM SG: "Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on the purpose or value of the content, and include words that users are likely to search for. For example, in a short
description in an online help topic, provide users with enough information to determine whether the topic meets their
needs. You can use self-referential text in learning modules and tutorials."